Polyhedron/demo/Polyhedron/Scene_c3t3_item.cpp

```diff
@@ -28,8 +28,7 @@
 #include <CGAL/AABB_C3T3_triangle_primitive.h>
 #include <CGAL/Polygon_mesh_processing/orient_polygon_soup.h>
 #include <CGAL/Polygon_mesh_processing/polygon_soup_to_polygon_mesh.h>
-
-
+#include <QKeyEvent>
```

Comment:
To minimize the patch: remove this `#include <QKeyEvent>` and re-add the two empty lines.